enterprise/changelog.md

```diff
@@ -7,6 +7,7 @@ New releases are announced on the [reviewable-enterprise-announce mailing list](
 - See also the public [list of bugs](https://github.com/Reviewable/Reviewable/labels/bug) for Reviewable.
 
 #### Upcoming changes (min 1992.2986 GHE 2.12+)
+- Fix: don't allow a race condition to cause some values like mergeFlavor to flutter when code completion code is run.
```

Comment:
These notes are for external consumption, so make sure to use public names instead of internal ones when they differ.  In this case `mergeFlavor` -> `mergeStyle`.

Reply:
Good point.  Fixed.